include/networkit/io/NetworkitBinaryWriter.hpp

```diff
@@ -9,6 +9,7 @@
 
 #include <networkit/graph/Graph.hpp>
 #include <networkit/io/GraphWriter.hpp>
+#include <networkit/io/NetworkitBinaryGraph.hpp>
```

Comment:
This include was moved here (from `NetworKitBinaryWriter.cpp`); however, as far as I can see, it is not used in this header.